### PR TITLE
[M2HT1] - add basic markup components

### PR DESCRIPTION
**Pull request by rshv00**

### CODE REVIEW

File: task1-basic_markup/about.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        body {
+            max-width: 900px;
+            font-family: 'Open Sans', sans-serif;
+            font-weight: 400;
+            font-size: 18px;
+            margin-top: 40px;
+            margin-right: 40px;
+        }
```
Comment (Misha1Shevchuk) [inline task1-basic_markup/about.html:27]: margin with body it is not good.

File: task1-basic_markup/education.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            box-sizing: border-box;
+            margin: 0;
+            padding: 0;
+        }
+
+        body {
+            font-family: 'Open Sans', sans-serif;
+            font-weight: 400;
+            font-size: 18px;
+            margin-top: 40px;
```
Comment (Misha1Shevchuk) [inline task1-basic_markup/education.html]: margin with body it is not good

File: task1-basic_markup/education.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        }
+    </style>
+</head>
+<body>
+<div class="content">
+    <h3>Education</h3>
+    <div class="education-box">
+        <div class="timeline-item">
+            <div class="timeline-left">
+                <div class="vertical-line-before-first">
```
Comment (Misha1Shevchuk) [inline task1-basic_markup/education.html]: try to use css selector :before

File: task1-basic_markup/side-nav.html [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+            font-weight: 700;
+        }
+
+        .nav-list-item {
+            position: relative;
+            padding: 10px 0 10px 20px;
+            border-bottom: 1px dashed gray;
+        }
+
+        .nav-list-item > a {
```
Comment (Misha1Shevchuk) [inline task1-basic_markup/side-nav.html]: better add personal class for "a" tag

File: task1-basic_markup/side-nav.html [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+            display: flex;
+            flex-direction: row;
+            filter: invert(86%) sepia(4%) saturate(40%) hue-rotate(313deg) brightness(100%) contrast(90%);
+        }
+
+        .nav-list-item > a > img {
```
Comment (Misha1Shevchuk) [inline task1-basic_markup/side-nav.html]: better add personal class for "img" tag

File: task1-basic_markup/side-nav.html [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+<body>
+<div class="container">
+    <div class="navigation">
+        <div class="user-brief-info">
+            <img class="profile-photo" src="assets/profile_photo.jpg" alt="Profile photo">
+            <span class="name">John Doe</span>
+        </div>
+        <nav class="navbar">
+            <ul class="navbar-list">
+                <li class="nav-list-item">
```
Comment (rshv00) [inline task1-basic_markup/side-nav.html]: better to print list items by using JS

**Comment by rshv00** [comment]
check responsivenness

File: task1-basic_markup/side-nav.html [hunk 7]
```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
```
Comment (rshv00) [inline task1-basic_markup/side-nav.html:1]: check responsiveness

File: task1-basic_markup/about.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+        a {
+            color: #000000;
+            text-decoration: none;
+        }
+
+        h3 {
+            color: #29AB87;
```
Comment (rshv00) [inline task1-basic_markup/about.html:32]: should be #fff

**Pull request merged**